.eslintrc.cjs

```diff
@@ -0,0 +1,108 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2024: true,
```

Comment:
The `es2024` environment setting is not standard. The latest available ECMAScript version is `es2021`. Consider using `es2021` unless you have a specific reason to use a future version.